common/src/main/java/com/microsoft/hydralab/performance/parsers/WindowsMemoryResultParser.java

```diff
@@ -53,6 +59,15 @@ public PerformanceTestResult parse(PerformanceTestResult performanceTestResult)
 
                         parsedData.getProcessIdProcessNameMap().put(processId, processName);
                         parsedData.getProcessIdWindowsMemoryMetricsMap().put(processId, windowsMemoryMetrics);
+
+                        averagedData.getProcessIdProcessNameMap().putIfAbsent(processId, processName);
+                        averagedData.getProcessIdWindowsMemoryMetricsMap().putIfAbsent(processId,
+                                new WindowsMemoryParsedData.WindowsMemoryMetrics());
+                        averagedData.getProcessIdWindowsMemoryMetricsMap().get(processId).accumulate(
```

Comment:
If we accumulate all the memory data, will it cause overflow? I suggest we calculate the average data simultaneously.

Reply:
Agreed, especially for those tests which run for very long periods of time.
Thanks. Updated.